Core/tooling/tslint-rules/noDirectAjaxRule.ts

```diff
@@ -0,0 +1,28 @@
+import * as ts from "typescript";
+import * as tslint from "tslint";
+
+const DO_NOT_USE_AJAX = `Direct ajax calls don't belong here.
+
```

Comment:
Can remove this rule - there is no network support in iOS JavaScriptCore anyway.